turtlebot2_amcl/launch/turtlebot_amcl.py

```diff
@@ -0,0 +1,132 @@
+    ld.add_process(
+        cmd=[get_executable_path(package_name=package, executable_name='kobuki_node')],
+        name='kobuki_node',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'astra_camera'
+    ld.add_process(
+        cmd=[
+            get_executable_path(package_name=package, executable_name='astra_camera_node'),
+            '-dw', '320', '-dh', '240', '-C', '-I'],
+        name='astra_camera_node',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'depthimage_to_laserscan'
+    ld.add_process(
+        cmd=[
+            get_executable_path(
+                package_name=package, executable_name='depthimage_to_laserscan_node')],
+        name='depthimage_to_laserscan_node',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'tf2_ros'
+    ld.add_process(
+        # The XYZ/Quat numbers for base_link -> camera_rgb_frame are taken from the
+        # turtlebot URDF in
+        # https://github.com/turtlebot/turtlebot/blob/931d045/turtlebot_description/urdf/sensors/astra.urdf.xacro
+        cmd=[
+            get_executable_path(
+                package_name=package, executable_name='static_transform_publisher'),
+            '-0.087', '-0.0125', '0.287',
+            '0', '0', '0', '1',
+            'base_link',
+            'camera_rgb_frame'
+        ],
+        name='static_tf_pub_base_rgb',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'tf2_ros'
+    ld.add_process(
+        # The XYZ/Quat numbers for camera_rgb_frame -> camera_depth_frame are taken from the
+        # turtlebot URDF in
+        # https://github.com/turtlebot/turtlebot/blob/931d045/turtlebot_description/urdf/sensors/astra.urdf.xacro
+        cmd=[
+            get_executable_path(
+                package_name=package, executable_name='static_transform_publisher'),
+            '0', '0.0250', '0',
+            '0', '0', '0', '1',
+            'camera_rgb_frame',
+            'camera_depth_frame'
+        ],
+        name='static_tf_pub_rgb_depth',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'joy'
+    ld.add_process(
+        cmd=[get_executable_path(package_name=package, executable_name='joy_node')],
+        name='joy_node',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'teleop_twist_joy'
+    ld.add_process(
+        cmd=[get_executable_path(package_name=package, executable_name='teleop_node')],
+        name='teleop_node',
+        exit_handler=restart_exit_handler,
+    )
+    turtlebot2_amcl_share = get_package_share_directory('turtlebot2_amcl')
+    map_path = os.path.join(turtlebot2_amcl_share, 'examples', 'osrf_map.yaml')
+    if args.map:
+        map_path = args.map
+    package = 'map_server'
+    ld.add_process(
+        cmd=[get_executable_path(package_name=package, executable_name='map_server'), map_path],
+        name='map_server',
+    )
+    ld.add_process(
+        cmd=[
+            'amcl', '--use-map-topic',
```

Comment:
amcl gets installed to the path at the moment because of https://github.com/ament/ament_cmake/issues/96